packages/standard-server-fetch/src/event-source.ts

```
@@ -0,0 +1,123 @@
+  ErrorEvent,
+  EventDecoderStream,
+  getEventMeta,
+  isEventMetaContainer,
+  parseEmptyableJSON,
+  UnknownEvent,
+  withEventMeta,
+} from '@orpc/server-standard'
+
+export function toEventIterator(
+  stream: ReadableStream<Uint8Array>,
+): AsyncGenerator<JsonValue | void, JsonValue | void, void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)